mgmtd/mgmt_fe_adapter.c

```diff
@@ -1443,20 +1443,14 @@ static void fe_adapter_handle_rpc(struct mgmt_fe_session_ctx *session,
 		return;
 	}
 
-	if (snode->nodetype == LYS_RPC)
-		clients =
-			mgmt_be_interested_clients(xpath,
-						   MGMT_BE_XPATH_SUBSCR_TYPE_RPC);
-	else if (snode->nodetype == LYS_ACTION)
-		clients =
-			mgmt_be_interested_clients(xpath,
-						   MGMT_BE_XPATH_SUBSCR_TYPE_CFG);
-	else {
+	if (snode->nodetype != LYS_RPC && snode->nodetype != LYS_ACTION) {
 		fe_adapter_send_error(session, req_id, false, -EINVAL,
 				      "Not an RPC or action path: %s", xpath);
 		return;
 	}
 
+	clients = mgmt_be_interested_clients(xpath,
+					     MGMT_BE_XPATH_SUBSCR_TYPE_RPC);
```

Comment:
don't need `MGMT_BE_XPATH_SUBSCR_TYPE_CFG` anymore - `TYPE_RPC` for all clients ?

Reply:
There are 4 types actually (CFG, OPER, NOTIFY, RPC) that are used in other contexts.

Reply:
Perhaps I misunderstood the comment though. The point of the change was to consolidate RPC and Action into the one "I'm interested" map registry for all "actions", and not to have 1 map for RPCs and then have 2 different meanings for the config map registry. This change removes that double use of config so that config maps are once again solely for indicating interest in config changes.

Reply:
yes, thanks - that's what I was asking about.